public/app/features/search/service/utils.ts

```diff
@@ -38,11 +40,18 @@ function delay(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
-export function getIconForKind(kind: string, isOpen?: boolean): IconName {
+export function getIconForItem(itemOrKind: string | DashboardViewItemWithUIItems, isOpen?: boolean): IconName {
```

Comment:
At that point, I think it's much of a muchness, and I'm happy to stick with the current implementation.

I think it would be much better if we had a DashboardViewItem for this in Search, but we don't at the moment so 🤷

Reply:
i did this! 👍